File: tests/go/cases/case_207_dead_sink_decoy_01.go
package escape_tests

var retainedCase207 = []map[string]string{}

func Case207DeadSinkDecoy01(input string) string {
	raw := input
	if raw == "" {
		raw = "sample"
	}
	payload := map[string]string{
		"task": "dead_sink_decoy_01",
		"entity": "extreme",
		"stage": "stress",
		"input": raw,
		"checksum": "dead_sink_decoy_01:" + raw,
	}
	if 1 == 0 {
		retainedCase207 = append(retainedCase207, payload)
	}
	// SAFE: dead branch never executes sink.
	return payload["checksum"]
}

File: tests/nodejs/cases/case_207_dead_sink_decoy_01.js
/**
 * Task case 207: dead_sink_decoy_01 deep stress pattern.
 */

const retainedCache = new Map();
const retainedAudit = [];
const retainedHandlers = [];

function case207DeadSinkDecoy01(input) {
  const taskName = 'dead_sink_decoy_01';
  const raw = input || 'sample';
  const payload = {
    task: taskName,
    entity: 'extreme',
    stage: 'stress',
    input: raw,
    checksum: `${taskName}:${raw.length}`,
  };
  if (1 === 2) { retainedAudit.push(payload); }
  // SAFE: dead branch only.
  return payload.checksum;
}

module.exports = { case207DeadSinkDecoy01 };
